packages/venia-ui/lib/components/Header/searchTrigger.js

```diff
@@ -27,14 +29,15 @@ const SearchTrigger = props => {
                 defaultMessage: 'Search'
             })}
             onClick={handleClick}
+            ref={ref}
         >
             <Icon src={SearchIcon} />
             <span className={classes.label}>
                 <FormattedMessage id={'Search'} />
```

Comment:
You didn't add this, but while you're in here can you fix the warning:

[![Image from Gyazo](https://i.gyazo.com/f534ea206066ba7b3a932825053650a9.png)](https://gyazo.com/f534ea206066ba7b3a932825053650a9)

Reply:
https://github.com/magento/pwa-studio/pull/2685/commits/a2e1055bce4061c684505e4fd13403d1a6180ff6